log/wrapper_example_test.go

```diff
@@ -18,6 +18,19 @@ type ExtendedLogWrapper struct {
 	log.Wrapper
 }
 
+var st = metricsbp.NewStatsd(
+	context.Background(),
+	metricsbp.Config{
+		// This is to make sure that when we read the metricsbp statsd buffer out
+		// manually later in the example they are not already sent to a blackhole
+		// sink.
+		// This is NOT needed in production code as in production code the sink
+		// shall be configured as the actual statsd collector and not read out
+		// manually like in this example.
+		NoBlackhole: true,
```

Comment:
what about `ForceInMemoryBuffer` or `BufferInMemoryForTesting`?

Can this be an unexported struct field?  (I assume it's exported since you think other packages need this too)

Reply:
This must be exported, for example the line you commented on is outside of the original package.

Will rename it to `BufferInMemoryForTesting`.